src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double height;
+    private double aBase;
+    private double bBase;
+
+    public IsoscelesTrapezoid(double height, double aBase, double bBase, String color) {
+        super(color);
+        this.height = height;
+        this.aBase = aBase;
+        this.bBase = bBase;
+    }
+
+    public double getbBase() {
+        return bBase;
+    }
+
+    public void setbBase(double bBase) {
+        this.bBase = bBase;
+    }
+
+    public double getaBase() {
+        return aBase;
+    }
+
+    public void setaBase(double aBase) {
+        this.aBase = aBase;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
+        return ((aBase + bBase) * height) / 2;
```

Comment:
The method name 'area' does not clearly convey that it calculates the area of the figure. Consider renaming it to 'calculateArea' to be more descriptive.